e2e/test/prerequisites/E2ETestsSetup/e2eTestsSetup.ps1

```diff
@@ -554,36 +556,6 @@ az iot dps enrollment create `
     --certificate-path $individualDeviceCertPath `
     --output none
 
-# The Service Principal takes a while to get propogated and if a different endpoint is hit before that, trying to grant a permission will fail.
-# Adding retries so that we can grant the permissions successfully without re-running the script.
-Write-Host "`nGranting $logAnalyticsAppId Reader role assignment to the $ResourceGroup resource group."
```

Comment:
az ad sp create-for-rbac ensures we don't need any retry logic. So removing this logic now.